addons/mail/models/mail_render_mixin.py

```diff
@@ -208,7 +208,11 @@ def _prepend_preview(self, html, preview):
 
     def _has_unsafe_expression(self):
         for template in self.sudo():
+            Model = self.env[template.model] if template.model else None
+            model_defaults = Model is not None and hasattr(Model, '_mail_template_default_values') and Model._mail_template_default_values() or {}
             for fname, field in template._fields.items():
+                if template[fname] == model_defaults.get(fname, ''):
+                    continue
```

Comment:
@tde-banana-odoo if you'd like to give it a look, it's fairly simple as a global fix and should be safe assuming we don't set unsafe defaults (don't think there's any reason to do that...)

Reply:
Shouldn't it be overriden on mail.template model ? Other model inheriting from the render mixin don't need that ...